jupyter-scipy/rockcraft.yaml

```diff
@@ -91,20 +115,27 @@ parts:
     plugin: nil
     source: https://github.com/kubeflow/kubeflow
     source-type: git
-    source-tag: v1.8-branch # upstream branch
+    source-tag: v1.10.0-rc.0
     build-packages:
       - lsb-release
       - wget
     build-environment:
+      - JUPYTERLAB_VERSION: "4.2.5"
+      - JUPYTER_VERSION: "7.2.2"
       - MINIFORGE_ARCH: "x86_64"
-      - MINIFORGE_VERSION: "4.10.1-4"
-      - PIP_VERSION: "21.1.2"
-      - PYTHON_VERSION: "3.8.10"
+      - MINIFORGE_VERSION: "24.7.1-2"
+      - PIP_VERSION: "24.2"
+      - PYTHON_VERSION: "3.11.10"
       - HOME: "/home/jovyan"
+      - HOME_TMP: /tmp_home/jovyan
```

Comment:
question: is this needed? iiuc in upstream they use this to copy contents from `HOME_TMP` to `HOME`  at runtime

Reply:
I would rather keep it ... I checked again the environment variables of the process in upstream container and I can see there `HOME_TMP`